src/Util/SupportedAddonsLoader.php

```diff
@@ -14,9 +14,9 @@ class SupportedAddonsLoader extends ApiLoader
      */
     public function getAddonNames()
```

Comment:
Update PHPDoc. Nothing to do with addons.silverstripe.org anymore.

Reply:
Updated